### PR TITLE
Added truths (and a lie)

### PR DESCRIPTION
**Pull request by kasheehy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ Write down two truths and one lie below.
 
 Once you're done, submit a pull request (do NOT commit directly to the main branch)! We'll try to guess the lie before we merge your branch with the main branch.
 
-YOUR NAME HERE:
-1. A truth (or a lie)
-2. A lie (or a truth)
-3. Another truth (or your lie)
+Kirsten:
+1. I was burned in a terrible picle accident
+2. I was hit by a taco truck (and got a free taco)
```
Comment (CBrothers96) [inline README.md:10]: I think the taco tasted like a lie bc it is

Comment (ablonder) [inline README.md:10]: A lie?

Comment (izzymolly) [inline README.md:10]: I think this was is a lie!

Comment (jcbjohnson) [inline README.md:10]: I have been hit by three taco trucks. This is true.

**Pull request merged**